project.html

```diff
@@ -0,0 +1,176 @@
+                    <div class="hexagon-layer">
+                    </div>
+                </div>
+            </div>
+            <div class="hexagon1">
+                <div class="hexagon-layer-11">
+                    <div class="hexagon-layer1">
+                    </div>
+                </div>
+            </div>
+            <div class="hexagon2">
+                <div class="hexagon-layer-12">
+                    <div class="hexagon-layer2">
+                    </div>
+                </div>
+            </div>
+            <div class="hexagon3">
+                <div class="hexagon-layer-13">
+                    <div class="hexagon-layer3">
+                    </div>
+                </div>
+            </div>
+            <br>
+            <div class="hexagon4">
+                <div class="hexagon-layer-14">
+                    <div class="hexagon-layer4">
+                    </div>
+                </div>
+            </div>
+            <div class="hexagon5">
+                <div class="hexagon-layer-15">
+                    <div class="hexagon-layer5">
+                    </div>
+                </div>
+            </div>
+            <div class="hexagon6">
+                <div class="hexagon-layer-16">
+                    <div class="hexagon-layer6">
+                    </div>
+                </div>
+            </div>
+            <div class="hexagon7">
+                <div class="hexagon-layer-17">
+                    <div class="hexagon-layer7">
+                    </div>
+                </div>
+            </div>
+            <div class="hexagon8">
+                <div class="hexagon-layer-18">
+                    <div class="hexagon-layer8">
+                    </div>
+                </div>
+            </div>
+            <br>
+            <div class="hexagon9">
+                <div class="hexagon-layer-19">
+                    <div class="hexagon-layer9">
+                    </div>
+                </div>
+            </div>
+            <div class="hexagon10">
+                <div class="hexagon-layer-110">
+                    <div class="hexagon-layer10">
+                    </div>
+                </div>
+            </div>
+            <div class="hexagon11">
+                <div class="hexagon-layer-111">
+                    <div class="hexagon-layer11">
+                    </div>
+                </div>
+            </div>
+            <div class="hexagon12">
+                <div class="hexagon-layer-112">
+                    <div class="hexagon-layer12">
+                    </div>
+                </div>
+            </div>
+
+        </section>
+        <section class="info">
+            <div class="smit-foto">
+                <div class="smit-foto-layer-1">
+                    <div class="smit-foto-layer">
+                    </div>
+                </div>
+            </div>
+            <div class="smit-line"></div>
+            <div class="smit"></div>
```

Comment:
Dont use  backgrond-image for creating text

Reply:
Also for creating this info, you can use tag article. It is more semantic. 
`<section><article>...</article></section>`